### PR TITLE
Fix Sofa Armrest Colors

### PR DESCRIPTION
**Pull request by Geevies**
* Fixed sofa armrests being the wrong colour.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
!bugfix

**Review by alsoandanswer — APPROVED** [review]

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 1]
```diff
@@ -29,12 +29,12 @@
 	generate_overlay_cache(material, CACHE_TYPE_OVER, ABOVE_HUMAN_LAYER) // Generate overlay for backrest
 	// Padding overlay.
 	if(padding_material)
-		generate_overlay_cache(padding_material, CACHE_TYPE_PADDING_OVER, ABOVE_HUMAN_LAYER, TRUE) // Generate padding overlay for backrest
+		generate_overlay_cache(padding_material, CACHE_TYPE_PADDING_OVER, ABOVE_HUMAN_LAYER + 0.01, TRUE) // Generate padding overlay for backrest
```
Comment (FluffyGhoster) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm:32]: No, use a defined layer without offsets if possible

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 2]
```diff
@@ -29,12 +29,12 @@
...
 
 	if(buckled)
 		generate_overlay_cache(material, CACHE_TYPE_ARMREST, ABOVE_HUMAN_LAYER) // Generate armrests
 		if(padding_material)
-			generate_overlay_cache(padding_material, CACHE_TYPE_PADDING_ARMREST, ABOVE_HUMAN_LAYER, TRUE) // Generate padding overlay for armrest
+			generate_overlay_cache(padding_material, CACHE_TYPE_PADDING_ARMREST, ABOVE_HUMAN_LAYER + 0.01, TRUE) // Generate padding overlay for armrest
```
Comment (FluffyGhoster) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm:37]: Same

**Pull request merged**